notebooks/prompt_customization_for_Anthropic.ipynb

```diff
@@ -48,11 +48,7 @@
       "cell_type": "code",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #15.    prompt_builder = ChatPromptBuilder(template=messages)
</pre><p> </p><p>If you want to avoid the warning, using <code>ChatPromptBuilder(template=messages, required_variables="*")</code> may help</p><p><a href="https://docs.haystack.deepset.ai/reference/builders-api#chatpromptbuilder__init__" target="_blank">https://docs.haystack.deepset.ai/reference/builders-api#chatpromptbuilder__init__</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-cookbook/pull/191/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>